feature/bgp/linkbandwidth_community/otg_tests/linkbandwidth_aggregation/README.md

```diff
@@ -0,0 +1,84 @@
+# RT-7.6: BGP Link Bandwidth Community - Aggregation
+
+## Summary
+
+This test verifies Link-bandwidth (LBW) extended community aggregation feature by DUT.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+    ```
+                                          |         |
+    [ 64x eBGP ] --- [ ATE Port 1 ] ----  |   DUT   | ---- [ ATE Port 2 ]
+                                          |         |
+    ```
+
+### RT-7.6.1: Verify LBW aggregation to eBGP peer
+
+* Configure DUT with routed ports on DUT.
+* Configure 64x eBGP peers on ATE Port 1 interface in peering group UPSTREAM.
+* Configure a single eBGP peer on ATE Port 2 interface in peering group DOWNSTREAM.
```

Comment:
Style suggestion to place the pre-requisite configuration into a common section called Test Bed Setup Configuration (before the 7.6.1,2 subtests).   Pushing this config is not considered part of the 'test criteria', it's just a prerequisite.

Reply:
ATE configuration should also be moved to the test bed setup configuration.